internal/attestation/crafter/crafter.go

```diff
@@ -293,7 +293,9 @@ func gracefulGitRepoHead(path string) (*HeadCommit, error) {
 
 	remotes, err := repo.Remotes()
 	if err != nil {
-		return nil, fmt.Errorf("getting remotes: %w", err)
+		// go-git does an additional validation that the branch is pushed upstream
+		// we do not care about that use-case, so we ignore the error
+		return c, nil
```

Comment:
I know this is swallowing other errors, but I decided to go this way since

- That error type is not exposed in the upstream library
- We really shouldn't block on remotes lookups for any reason.

Reply:
I understand the error, but I can see that the function `repo.Remotes()` can return more than one type of error. I think you want to swallow only if err is `ErrRemoteNotFound` or equivalent, am I right?

Reply:
That's right, that's what I tried to explain in my comment. "I know this is swallowing other errors"

I just thought it was not worth inspecting them as long as we log them (see the logger.Warn), and I do not see a reason why we should fail in any circumstance. 

In any case, let me revisit it.

Thanks!

Reply:
Ok so you are ubersure that no matter what happens, we can keep going.
Gotcha, I'll approve now that I'm sure <3

Reply:
ptal at the change now.